app/services/TwoStepVerification.scala

```diff
@@ -39,37 +38,43 @@ trait TwoStepVerification {
 
   def twoStepVerificationThrottle: TwoStepVerificationThrottle
 
-  case class Biz2SVRule(name: String, conditions: List[Condition])
+  def twoStepVerificationRuleFactory = new TwoStepVerificationUserSegmentFactory {}
```

Comment:
Is there any reason why there's no companion object for `TwoStepVerificationUserSegmentFactory`?